File: 은지/LV2/이진 변환 반복하기.js
function solution(s) {
  var zero = 0
  var cnt = 0
  while (s !== "1") {
    cnt += 1
    var cnt_zero = 0
    var cnt_one = 0
    for (let x of s) {
      if (x === "1") {
        cnt_one += 1
      } else {
        cnt_zero += 1
      }
    }
    zero += cnt_zero
    s = cnt_one.toString(2)
  }
  var answer = [cnt, zero]
  return answer
}

File: 은지/LV2/이진 변환 반복하기.py
# 10진수 => 2진수
def two(x):
    if x < 2:
        return x
    else:
        return x % 2 + two(x//2)*10

def solution(s):
    zero = cnt = 0
    # s가 1이 되면 끝!
    while s != "1":
        # 반복 횟수
        cnt += 1
        # 0 개수
        zero += s.count("0")
        # 1의 개수를 이진수로 변경
        s = str(two(s.count("1")))
    answer = [cnt, zero]
    return answer